client/src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { Callout } from 'plaid-threads/Callout';
+import { IconButton } from 'plaid-threads/IconButton';
+import { CloseS2 } from 'plaid-threads/Icons/CloseS2';
+
+import useErrors from '../services/errors';
+
+//  Allows user to input their personal assets such as a house or car.
+
+export default function ErrorMessage() {
+  const [show, setShow] = useState(false);
+  const [message, setMessage] = useState('');
+  const { error, resetError } = useErrors();
+
+  useEffect(() => {
+    const errors = [
```

Comment:
what is this array of `error_codes`? are they just ones you expect to see / ones with error messages?

Reply:
These are the ones that will need to prompt some sort of user behavior.